### PR TITLE
[WIP] Remove organization from the apb spec.

### PR DESCRIPTION
**Pull request by djzager**
- Update the apb cli to remove the organization argument.
- Update the apb engine so we do not put the organization into the apb.yml
- specification.
- Update documentation to remove the organization.

### CODE REVIEW

**Comment by djzager** [comment]
# Testing

## Build

```
(apb) ➜  ansible-playbook-bundle git:(remove-org) ✗ rm -rf build 
(apb) ➜  ansible-playbook-bundle git:(remove-org) ✗ pip install -r src/requirements.txt && python setup.py install
...
Finished processing dependencies for apb==0.2.2
```
## Help screen

```
(apb) ➜  ansible-playbook-bundle git:(remove-org) ✗ apb --help                                                    
usage: apb [-h] [--debug] [--project BASE_PATH]
           {init,help,prepare,push,bootstrap,list,remove,build} ...

APB tooling for assisting in building and packaging APBs.

optional arguments:
  -h, --help            show this help message and exit
  --debug               Enable debug output
  --project BASE_PATH, -p BASE_PATH
                        Specify a path to your project. Defaults to CWD.

subcommand:
  {init,help,prepare,push,bootstrap,list,remove,build}
    init                Initialize the directory for APB development
    help                Display this help message
    prepare             Prepare an ansible-container project for APB packaging
    push                Push local APB spec to an Ansible Service Broker
    bootstrap           Tell Ansible Service Broker to reload APBs from the
                        container repository
    list                List APBs from the target Ansible Service Broker
    remove              Remove APBs from the target Ansible Service Broker
    build               Build and package APB container
```

## Org arg no longer supported

```
(apb) ➜  ansible-playbook-bundle git:(remove-org) ✗ apb init my_apb --async=optional --bindable --org foo_org     
usage: apb [-h] [--debug] [--project BASE_PATH]
           {init,help,prepare,push,bootstrap,list,remove,build} ...
apb: error: unrecognized arguments: --org foo_org
```

## APB.yml no longer includes org

```
(apb) ➜  ansible-playbook-bundle git:(remove-org) ✗ apb init my_apb --async=optional --bindable              
Exception occurred! ERROR: Project directory: [/home/dzager/Workspace/ansible-playbook-bundle/my_apb] found and force option not specified
(apb) ➜  ansible-playbook-bundle git:(remove-org) ✗ cat my_apb/apb.yml 
name: my_apb
image: my_apb
description: This is a sample application generated by apb init
bindable: True
async: optional
metadata:
  displayName: my_apb
plans:
  - name: default
    description: This default plan deploys my_apb
    free: True
    metadata: {}
    parameters: []
```

**Comment by djzager** [comment]
I'm going to mark this as WIP so I can build a new broker and test with an apb I built with these changes.

**Comment by dymurray** [comment]
@djzager Could we just go ahead and remove the `image:` field while we're at it? Is `name` somehow used differently on the broker side?

File: src/apb/engine.py [hunk 1]
```diff
@@ -473,20 +473,12 @@ def cmdrun_init(**kwargs):
     else:
         app_name = apb_name
 
-    organization = kwargs['org']
-    if organization is None:
-        if len(apb_tag_arr) > 1:
```
Comment (dymurray) [inline src/apb/engine.py:478]: Since we no longer need `apb_tag_arr` I suggest on lines 469-470 we instead have:
apb_name = kwargs['tag'].

**Comment by jmrodri** [comment]
Removing the image seems like a valid idea since having duplicate information is bad. I don't think removing image is trivial. I propose we close this PR. Then create a new proposal for removing image from the spec (which will also remove the org). This has some big ramifications on both the apb and the broker side.

1) removing image will bring up the apb versioning discussion again
2) the broker makes use of the image name at the moment, removing from spec we'll have to generate it

**Pull request closed without merging**